internal/app/app.go

```diff
@@ -73,24 +73,32 @@ type SlimConfig struct {
 
 // Config wraps the properties of app configuration.
 type Config struct {
-	AppName     string                 `toml:"app,omitempty"`
-	Build       *Build                 `toml:"build,omitempty"`
-	HttpService *HttpService           `toml:"http_service,omitempty"`
-	VM          *VM                    `toml:"vm,omitempty"`
```

Comment:
sorry I am new here. Why is that VM is not needed anymore?

Reply:
Setting the port to 8080 should have always been explicitly set on Rails apps. It's just that 8080 is the default for Nomad apps.

`VM` was prematurely added by me as a way to specify VM size in machine config. We backed out of that since we probably need to work out process groups first, then see how VM sizes might apply to those.